### ISSUE TITLE
Fetch Emoji data from Emojibase instead of a local lookup

### ISSUE THREAD

**Issue by sr229** [Earu/EasyChat#104]
[Emojibase](https://emojibase.dev/) maintains a reliable dataset for emoji lookups and uptime can be ensured from their endpoints, with the option to host it on your own if needed. This should eliminate the manual task to update the lookup table everytime there's an update on the Emoji spec.

**Comment by Earu** [comment]
I'm not actually sure whats there to change here. As of right now the lookup table is in this repo, its just downloaded by each client so that they dont constantly re-download it. You can find it [here](https://raw.githubusercontent.com/Earu/EasyChat/master/external_data/twemojis.json). As for emojibase or whichever other dataset the problem is the lookup is not the same as discord which was what we aimed for with this.

TL;DR I don't see the point of changing this at all

**Comment by Earu** [comment]
As mentioned previously I don't really see the point of this, closing the issue for now, open a new one if there is a need.
